src/status.js

```diff
@@ -0,0 +1,8 @@
+const Status = {
+    excludedDomain: "excludedDomain",
+    pronounSpecs: "pronounSpecs",
+    mentionsGender: "mentionsGender",
+    replacedPronouns: "replacedPronouns",
+    noGenderedPronouns: "noGenderedPronouns"
+};
```

Comment:
It seems like a waste of a dictionary to have the values identical to the keys. I wonder if you it would make sense put the status messages here, as the dictionary values, rather than in the `popup.js` code.

Reply:
I think it makes sense for the popup to be responsible for the status messages it displays. The popup needs to get a status (like 'mentionsGender') to tell it which interface items to show or hide. It would be more cumbersome to try and match a long string, and unnecessarily obtuse to use numerical status codes.

Reply:
Right, I think I'm getting muddled by JavaScript's object/dictionary duality. I'm willing to leave responsibility for the messages in the popup (at least for now).